### PR TITLE
Elewc 2316

### PR DESCRIPTION
**Pull request by loahou04**

### CODE REVIEW

**Comment by loahou04** [comment]
@JasonLantz why would we want onKeyUp rather on keydown?  Also don't we still want keyup to be on the document level?  Though the focus does seem like it could be at the component level because they'll still be focused on the component.

**Comment by JasonLantz** [comment]
could be an onKeyDown.... if you use onKeyDown on the level of the dropdown then the event listeners are scoped to the dropdown which unmounts (and takes the eventlisteners with it) with a selection from the dropdown...

**Comment by loahou04** [comment]
Ya we do unmount as well on componentWillUnmount.  but i agree with you
document level events aren't great.  i can see if i change it to component
level if it will still work properly...

**Comment by JasonLantz** [comment]
it's fine... just push the code.

**Comment by JasonLantz** [comment]
looks like the CI is failing on your namming of drop down item....
screen shot 2017-08-04 at 2 48 40 pm

File: demo/demoPages/DropdownPage.js [hunk 1]
```diff
@@ -1,105 +1,131 @@
...
-              presentationText="label"
-              list={simpleList}
-              mobileTitle="Mobile title"
-            />`}
+                dropdownControlLabel="Dropdown open"
+                mobileTitle={mobileTitle}
+                changeHandler={(item) => {
+                  console.log(item);
+                }}
+                targetElement={
```
Comment (mhomolak) [inline demo/demoPages/DropdownPage.js]: Weren't we avoiding using this? line 88 too

**Comment by loahou04** [comment]
yep forgot to update the documentation...

File: src/Dropdown/Dropdown.js [hunk 2]
```diff
@@ -58,130 +87,142 @@ export default class Dropdown extends Component {
...
-              transitionEnterTimeout={300}
-              transitionLeaveTimeout={300}>
-             {this.renderListItems()}
-            </CSSTransitionGroup> }
+        <div className="dropdown-container" onClick={this.toggleDropdown} onBlur={this.closeDropdown} >
+          {this.insertAnchor()}
+          <ul
+            role="menu"
+            id={`${this.props.id.replace(' ', '_')}-dropdown`}
+            ref={(dom) => { this.list = dom; }}
```
Comment (mhomolak) [inline src/Dropdown/Dropdown.js]: This ref is a leftover from previous implementation.

Comment (loahou04) [inline src/Dropdown/Dropdown.js]: no it is not.  I am using the list to go through the keydown events

**Comment by umahaea** [comment]
all changes tested. good to merge.

**Pull request merged**